util/linuxfw/nftables_runner.go

```diff
@@ -71,14 +71,17 @@ type nftable struct {
 type nftablesRunner struct {
 	conn *nftables.Conn
 	nft4 *nftable // IPv4 tables
-	nft6 *nftable // IPv6 tables
+	nft6 *nftable // IPv6 tables or nil if the system does not support IPv6
 
 	v6Available bool // whether the host supports IPv6
 }
 
 func (n *nftablesRunner) ensurePreroutingChain(dst netip.Addr) (*nftables.Table, *nftables.Chain, error) {
 	polAccept := nftables.ChainPolicyAccept
 	table := n.getNFTByAddr(dst)
```

Comment:
Would the intent calling this function be clearer if instead we updated the signature to return both a table and an error?